frontend/src/app/accountCreation/MfaEmailVerify/MfaEmailVerify.tsx

```diff
@@ -1,15 +1,15 @@
 import { MfaVerify } from "../MfaVerify/MfaVerify";
 
 interface Props {
-  email: string;
+  location: { state: { email: string } };
 }
 
 export const MfaEmailVerify = (props: Props) => (
   <MfaVerify
     hint={
       <>
-        We’ve sent an email to <b>{props.email}</b>. It will expire in 10
-        minutes.
+        We’ve sent an email to <b>{props.location?.state?.email || ""}</b>. It
+        will expire in 10 minutes.
```

Comment:
Is this expiry time provided by the backend (or okta) at all? Might be worth wiring it in here in case it changes. Or it might not be worth the effort 🤷‍♂️

Reply:
Hmmm, it's possibly provided by Okta, but I think it just defaults to 10 minutes and I don't know if we plan on changing it ever

Reply:
https://support.okta.com/help/s/question/0D50Z00008G7VcR/how-long-are-sms-one-time-passcode-valid?language=en_US

I think it actually defaults to 5 minutes, and it doesn't look like we can change that :/

Reply:
Aha! I will update the copy to 5  minutes then

Reply:
Fixed in 64b96ec